src/js/stores/MetronomeStore.js

```diff
@@ -165,6 +165,7 @@ class MetronomeStore extends EventEmitter {
           this.emit(METRONOME_JOB_SCHEDULE_UPDATE_SUCCESS, action.jobID);
           break;
         case REQUEST_METRONOME_JOBS_SUCCESS:
+          this.removeOldJobs(action.data);
```

Comment:
Can we add a small test for this?

Reply:
Yes, good idea, I'll add a test in `test-jobs.js` (our end to end system tests) 👍